package/src/components/MessageList/MessageList.tsx

```diff
@@ -619,7 +619,7 @@ const MessageListWithContext = <
             onThreadSelect={onThreadSelect}
             showUnreadUnderlay={showUnreadUnderlay}
             style={styles.messagePadding}
-            targetedMessage={targetedMessage === message.id}
```

Comment:
The boolean was sent from here and this was checked in Message.tsx, now instead of passing the boolean the value is itself passed and the condition is checked later. This is optional, should I remove the change? @vishalnarkhede